JitsiConference.js

```diff
@@ -248,6 +248,10 @@ JitsiConference.prototype._init = function(options = {}) {
             () =>
                 this.eventEmitter.emit(JitsiConferenceEvents.TALK_WHILE_MUTED));
     }
+
+    if (options.config.channelLastN) {
```

Comment:
options.config.channelLastN is not documented in int arguments. I think it may fit better to be configured in constructor and described like other options:
https://github.com/jitsi/lib-jitsi-meet/pull/489/commits/42253ee101c7ef0c1062d21dc678a11b4912e938#diff-40e35c919311b508dc5b45a7bf028f54L47